cli/running/running.go

```diff
@@ -655,33 +667,33 @@ func Start(cmdCtx *cmdcontext.CmdCtx, run *InstanceCtx) error {
 }
 
 // Stop the Instance.
-func Stop(run *InstanceCtx) error {
-	pid, err := getPIDFromFile(run.PIDFile)
+func Stop(pidFile, consoleSocket, processName string) error {
```

Comment:
In fact, you are redesigning a function to handle some abstract Process.
Maybe we should create a ProcessCtx and move this function to a separate module?

Reply:
I moved common functions for process management in `process_utils`.  `Stop` function for `running` with using `process_utils`:

```golang
// Stop the Instance.
func Stop(run *InstanceCtx) error {
	return process_utils.StopProcess(run.PIDFile, run.ConsoleSocket, "Instance")
}
```